cypress/integration/Feed_spec.js

```diff
@@ -0,0 +1,30 @@
+describe('Feed', () => {
+  beforeEach(() => {
+    cy.intercept("https://api.nytimes.com/svc/topstories/v2/home.json?api-key=GKUzDD1VY9ssjZ1AGusX3ci6AeoXCaSr", {fixture: 'nyt-article-fetch.json'})
```

Comment:
I didn't know how to use fixtures... thank you for integrating them, @najinl! So cool!